File: 5_multiple_inheritance.py
class vehical:
    def __init__(self, name, engin) :
        self.name = name
        self.engin = engin

    def topgirar(self, giar):
        print("vehical top giar", giar)
        
class Car:
    def __init__(self, model):
        self.model = model

    def modelmycar(self):
        print("my car model name ", self.model)

class privatcar(vehical, Car):
    def __init__(self, name, engin, model):
        vehical.__init__(self, name, engin)
        Car.__init__(self, model)

bmw = privatcar("banga cura" , 5, "ac13")

bmw.modelmycar()
bmw.topgirar(20)

File: 5_multiple_inheritance.cpp
#include<bits/stdc++.h>
using namespace std;
class Vehicle{
    public:
    string name;
    int engine;
    Vehicle(string name, int engine){
        this->name = name;
        this->engine = engine;
    }
    void topGear(int gear) {
        cout << "Vehicle top gear: " << gear << endl;
    }
};
class Car{
    public:
    string model;
    Car(string model){
        this->model = model;
    }
    void modelMyCar() {
        cout << "My car model name: " << model << endl;
    }
};

class privatecar: public Vehicle, public Car{
    public:
    privatecar(string name, int engine, string model): 
    Vehicle(name, engine), Car(model){}
};

int main(){
    privatecar Bmw("banga gari", 5, "Ac13");
    Bmw.modelMyCar();
    Bmw.topGear(20);

}